### ISSUE TITLE
Parse error using comparison operators in struct assignment

### ISSUE THREAD

**Issue by CamBurris** [cflint/CFLint#233]
I get the following errors when parsing a file that uses comparison operators in a struct assignment. You can also see that triggers a false positive missing semicolon error as well.

```
[ {
  "severity" : "ERROR",
  "id" : "MISSING_SEMI",
  "message" : "MISSING_SEMI",
  "category" : "CFLINT",
  "abbrev" : "MS",
  "locations" : [ {
    "fileName" : "onLoad.cfm",
    "function" : "",
    "column" : "1",
    "line" : "8",
    "message" : "End of statement(;) expected instead of }",
    "variable" : "",
    "expression" : "}"
  } ]
}, {
  "severity" : "ERROR",
  "id" : "PARSE_ERROR",
  "message" : "PARSE_ERROR",
  "category" : "CFLINT",
  "abbrev" : "PE",
  "locations" : [ {
    "fileName" : "onLoad.cfm",
    "function" : "",
    "column" : "10",
    "line" : "4",
    "message" : "Unable to parse",
    "variable" : "",
    "expression" : "no viable alternative at input '{\"a\":0<'"
  } ]
} ]
```

Sample code that triggers error:

```
<cfscript>
	public struct function onLoad() output="false" {
		local.a = {
			"a": 0 < 1
		};

		return local.a;
	}
</cfscript>
```

**Comment by ryaneberly** [comment]
thanks. fixed in CFParser
